### PR TITLE
chore: added how to create and use icon library

### PR DESCRIPTION
**Pull request by sarojgurung**

### CODE REVIEW

**Comment by sarojgurung** [comment]
is the install supposed to be
npm install @danielwang/aemo-gel-icon@1.1.0 ?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+### Use font icons by coding in HTML
+
+Gel icons use the `pi pi-{icon}` syntax such as pi pi-check following the reference from PrimeVue. A standalone icon can be displayed using an element like i or span.
+
+The icons are treated as fonts. So they can be resized by giving font-size styling to the icon class.
+
+```html
+<i class="pi pi-check" style="font-size: 2rem"></i>
+```
+
```
Comment (danielwang) [inline README.md:30]: please add steps of generating fonts files from svg by using the icomoon

**Review by danielwang — COMMENTED** [review]
please address my feedback

btw, this commit is not chore, it is docs

**Comment by danielwang** [comment]
> 

simply npm install @danielwang/aemo-gel-icon, without a version number, that should allow devs to install the latest version by default.

**Comment by danielwang** [comment]
>

File: README.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
-V1.1.1
\ No newline at end of file
+GEL icon library is a collection of web icons and system icons that are custom to AEMO. This section provides guidelines on how to use the current library and create additional icons when required.
+
+## Usage
+### Install
+Run the following command to install the package
+
+```bash
+npm install @danielwang/aemo-gel-icon@1.1.0
```
Comment (danielwang) [inline README.md:11]: change this to `npm install @danielwang/aemo-gel-icon`

**Review by danielwang — CHANGES_REQUESTED** [review]
please address my feedback

File: README.md (hunk 1 above)
Comment (sarojgurung) [reply README.md:30]: understood

**Review by danielwang — APPROVED** [review]

**Pull request merged**